src/Response/Embeds/AuthenticationResult.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Copyright (c) 2025 AltaPay
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is furnished
+ * to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+namespace Altapay\Response\Embeds;
+
+use Altapay\Response\AbstractResponse;
+
+class AuthenticationResult extends AbstractResponse
+{
+    /**
+     * @var string
```

Comment:
Why not making enum representations of it, so you do not need to parse strings?

Reply:
@emicha As we also support PHP 5.6 and do not validate responses in the SDK, do we still need an enum and to validate the values at the SDK level?

Reply:
You do not need to validate the responses if you want, but provide a hint of what the possible values are, otherwise the person will need to read up the documentation, which defeats the purpose of an SDK in the first place.

Reply:
@emicha I have updated the PHPDoc to add the possible values.